cmd/actuary/results/js/main.js

```diff
@@ -0,0 +1,174 @@
+window.onload=function(){
```

Comment:
Not sure of `cmd/actuary/results/` is the best place for this.

How about putting it on the act